jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
config is task env configuration files, so models are not in the dir.
Important is not a good word, all data are important.